cartography/intel/aws/lambda_function.py

```diff
@@ -36,6 +36,9 @@ def load_lambda_functions(neo4j_session, data, region, current_aws_account_id, a
     lambda.memory = {MemorySize},
     lambda.lastupdated = {aws_update_tag}
     WITH lambda
+    MATCH (owner:AWSAccount{id: {AWS_ACCOUNT_ID}})
+    MERGE (owner)-[r:RESOURCE]->(lambda)
```

Comment:
Need to add

```
ON CREATE SET r.firstseen = timestamp()
SET r.lastupdated = {aws_update_tag}
```

Also need to update the Lambda cleanup job to delete stale `:RESOURCE` relationships between Lambdas and their AWS accounts.